app/components/asset-reminder/team-members-selector.tsx

```diff
@@ -0,0 +1,107 @@
+import type { Prisma } from "@prisma/client";
+import { CheckIcon, UserIcon } from "lucide-react";
+import { Separator } from "~/components/shared/separator";
+import When from "~/components/when/when";
+import { useModelFilters } from "~/hooks/use-model-filters";
+import { tw } from "~/utils/tw";
+
+type TeamMembersSelectorProps = {
+  className?: string;
+  style?: React.CSSProperties;
+  error?: string;
+  defaultValues?: string[];
+};
+
+export default function TeamMembersSelector({
+  className,
+  style,
+  error,
+  defaultValues,
+}: TeamMembersSelectorProps) {
+  const {
+    items,
+    handleSearchQueryChange,
+    searchQuery,
+    handleSelectItemChange,
+    selectedItems,
+  } = useModelFilters({
+    selectionMode: "none",
+    defaultValues,
+    model: {
+      name: "teamMember",
+      queryKey: "name",
+      deletedAt: null,
+      userIsNotNull: true,
+    },
+    countKey: "totalTeamMembers",
```

Comment:
Here we should only get team members with roles admin and owner. We should not be showing base and self service users.